### ISSUE TITLE
UX of “Use for other thumbnails” button

### ISSUE THREAD

**Issue by lkiesow** [opencast/opencast-editor#832]
Looking at the “Use for other thumbnails” button in the thumbnail editor

- The button is too small for that text. There is almost no padding left and right
- The text doesn't really make sense, does it? It should be something like “Use (as thumbnail) for other video streams”. We don't use this for other thumbnails, we want to replace them with this image, don't we?

![Screenshot from 2022-10-18 13-58-27 (copy)](https://user-images.githubusercontent.com/1008395/196642357-99cdd5cb-706b-4650-bd0d-a9044fc0555d.png)

**Comment by cgruber-uos** [comment]
There are different thumbnails e.g. for "presenter" and "presentation", normally you are picking one image from the presenter / human video stream for "presenter" and an other slide snapshot from the slide / presentation stream. 

The button enables the user to use the e.g. selected "presenter" thumb for "others" thumbnails als, so all streams have the same thumbnail. So “Use for other thumbnails” is not totally precise but in the second run (vs. don't make me think :-) understandable in sense of “Use this as replacement for other thumbnails” 

Suggestions
- "Use this thumbnail for all streams"
- "Thumbnail for all streams"
- "Use this for all streams" [prefered]
- "Set for all streams"

**Comment by geichelberger** [comment]
I created the PR (https://github.com/opencast/opencast-editor/pull/840) before checking the issues 🙈 I changed it to `Use for other tracks`.
I think it's still not perfect, but using tracks instead of videos or streams would be consistent with the rest. The tab before already introduced the term.

Based on @cgruber-uos suggestions:

- Use for other tracks
- Use for all tracks
- Set for all tracks
